fixtures/hdfs/core-site.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+<property>
+  <name>fs.AbstractFileSystem.gs.impl</name>
+  <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS</value>
+  <description>The AbstractFileSystem for 'gs:' URIs.</description>
+</property>
+<property>
+  <name>fs.gs.project.id</name>
+  <value></value>
+  <description>
+    Optional. Google Cloud Project ID with access to GCS buckets.
+    Required only for list buckets and create bucket operations.
+  </description>
+</property>
+<property>
+  <name>google.cloud.auth.type</name>
+  <value>SERVICE_ACCOUNT_JSON_KEYFILE</value>
+  <description>
+    Authentication type to use for GCS access.
+  </description>
+</property>
+<property>
+  <name>google.cloud.auth.service.account.json.keyfile</name>
+  <value>/path/to/json_key_file</value>
```

Comment:
TODO (2/2): Need to substitute the path here to the json keyfile used to authenticate against GCS. I followed [these directions](https://cloud.google.com/storage/docs/authentication#service_accounts) to obtain a json keyfile for local dev - created a [service account](https://cloud.google.com/iam/docs/service-account-creds) and exported its credentials

So if we can store the whole json keyfile content inside the a github secret, we can create a JSON file dynamically using [this action](https://github.com/jsdaniell/create-json) and update the file path here

Reply:
ok, I got it. `op://services/gcs/credential` store the base64 content of a json key file. We can base64 decode it to a local path for test.

Reply:
Added action to create json key file for hdfs to load.